.github/workflows/rostemplate-gh-pages.yaml

```diff
@@ -1,12 +1,9 @@
 # Workflow derived from https://github.com/r-lib/actions/tree/master/examples
 # Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
 on:
-  workflow_dispatch:
   push:
     branches: [main, master]
     tags: ['*']
-  pull_request:
```

Comment:
will this not prevent the action from doing a test build of the package website on PRs?

Reply:
Yes but it's safe to assume that if the checks pass, the website will build.